src/solutions/ode_solutions.jl

```diff
@@ -14,6 +14,7 @@ struct ODESolution{T,N,uType,uType2,DType,tType,rateType,P,A,IType,DE} <: Abstra
   tslocation::Int
   destats::DE
   retcode::Symbol
+  residual::Function
```

Comment:
instead, it would probably better to add a `interpolation_residual(sol::ODESolution,t)` function.

Reply:
That the diff you were thinking of?